src/columnhandles.js

```diff
@@ -0,0 +1,273 @@
+
+    addRowAfterContainer.appendChild(addAfterButton);
+    const addRowAfterMarker = createElementWithClass(
+      'div',
+      'addRowAfterMarker'
+    );
+    addRowAfterContainer.appendChild(addRowAfterMarker);
+    this.dom.appendChild(addRowAfterContainer);
+  }
+
+  checkIfColHeader(view) {
+    const pos = this.getPos();
+    const resolvePos = view.state.doc.resolve(pos);
+    const rowStart = pos - resolvePos.parentOffset - 1;
+    const rowResolvedPos = view.state.doc.resolve(rowStart);
+
+    if (rowResolvedPos.parentOffset !== 0 || this.colHandle) return;
+
+    const colHandle = createElementWithClass('div', 'tableColHandle');
+    const colHandleButton = createElementWithClass(
+      'button',
+      'tableColHandleButton'
+    );
+    const buttonContent = createElementWithClass('span', 'buttonContent');
+    colHandleButton.appendChild(buttonContent);
+
+    colHandleButton.onclick = (e) => selectCol(e, view, this.getPos());
+
+    colHandle.appendChild(colHandleButton);
+    colHandle.contentEditable = false;
+
+    this.colHandle = colHandle;
+    this.dom.appendChild(colHandle);
+
+    this.colDragHandler = new ColDragHandler(
+      this.view,
+      this.colHandle,
+      document.body,
+      this.getPos,
+      this.dom
+    );
+
+    const addColAfterContainer = createElementWithClass(
+      'div',
+      'addColAfterContainer'
+    );
+    addColAfterContainer.contentEditable = false;
+
+    const addAfterButton = createElementWithClass('button', 'addAfterButton');
+    const addAfterButtonText = createElementWithClass('span', 'addButtonText');
+    addAfterButtonText.innerText = '+';
+    addAfterButton.appendChild(addAfterButtonText);
+    addAfterButton.appendChild(
+      createElementWithClass('div', 'addColButtonContent')
+    );
+
+    addAfterButton.onclick = () => {
+      addColBeforeButton(view, this.getPos());
+    };
+    addColAfterContainer.appendChild(addAfterButton);
+    const addColAfterMarker = createElementWithClass(
+      'div',
+      'addColAfterMarker'
+    );
+    addColAfterContainer.appendChild(addColAfterMarker);
+    this.colMarker = addColAfterContainer;
+    this.dom.appendChild(addColAfterContainer);
+
+    const tableAttrs = resolvePos.node(1).attrs;
+
+    // add sort and style only if headers allowed
+    if (tableAttrs.headers) {
+      this.dom.classList.add('colHeader');
+
+      const sortButton = createElementWithClass('button', 'sortColButton');
+      sortButton.contentEditable = false;
+
+      const sortedCol = tableAttrs.sort.col;
+      const colIndex = getColIndex(this.view.state, pos);
+
+      if (sortedCol === colIndex) {
+        sortButton.classList.add(
+          tableAttrs.sort.dir === 'down' ? 'sortedDown' : 'sortedUp'
+        );
+      }
+
+      sortButton.onclick = () => {
+        if (colIndex === null) return;
+
+        if (sortedCol !== colIndex || tableAttrs.sort.dir === 'up') {
+          sortColumn(view, colIndex, this.getPos(), 1);
```

Comment:
maybe just document that '1' is positive sort, and '-1' is negative sort? just because the `dir` can be `up` and `down`